components/ide/jetbrains/image/BUILD.js

```diff
@@ -0,0 +1,53 @@
+const ides = ["intellij", "goland", "pycharm", "phpstorm"];
+
+const packages = [];
+const deps = [];
+const generateIDEPackage = function (ideName, qualifier) {
+    let name = ideName + (qualifier === "stable" ? "" : "-" + qualifier);
+    let helmName = ideName + (qualifier === "stable" ? "" : "Latest");
+    let pkg = {
+        name,
+        type: "docker",
+        srcs: ["startup.sh", `supervisor-ide-config_${ideName}.json`],
+        deps: [
+            "components/ide/jetbrains/image/status:app",
+            `components/ide/jetbrains/image/download:${name}`,
```

Comment:
we should also generate them here to avoid boilerplate in https://github.com/gitpod-io/gitpod/blob/main/components/ide/jetbrains/image/download/BUILD.yaml